earth_enterprise/docs/landing_page/manual/answer/2981872.html

```diff
@@ -56,9 +56,6 @@
 </p></li>
 geselectassetroot [--lock] [--noprompt] [--unlock]
```

Comment:
I think that entry is actually unnecessary (cut-n-paste of usage without any context, possibly used by the original author to get the syntax right). Removed.